hdf5-sys/build.rs

```diff
@@ -622,8 +622,7 @@ impl Config {
         let mut vs: Vec<_> = (5..=21).map(|v| Version::new(1, 8, v)).collect(); // 1.8.[5-21]
         vs.extend((0..=8).map(|v| Version::new(1, 10, v))); // 1.10.[0-8]
         vs.extend((0..=2).map(|v| Version::new(1, 12, v))); // 1.12.[0-2]
-        vs.extend((0..=0).map(|v| Version::new(1, 13, v))); // 1.13.[0-0]
```

Comment:
Should we update version checks while we're at it? The latest versions are: 1.8.23, 1.10.10, 1.12.2, 1.14.1.

Reply:
I'll have a look at the changelog and update accordingly

Reply:
No major changes in those patch versions